.gitattributes

```diff
@@ -1,3 +1,50 @@
-tests/** export-ignore
-tests/ export-ignore
```

Comment:
I found that we don't need to specify `tests/**` and `tests/` in other to ignore the `tests` directory. Specifying the directory name without patterns works. I will like another person to verify this as I don't know if it was because of my git version.

Also, I updated the entry to be alphabetical a bit(not perfect though)

Reply:
Tested with:
```
mkdir dist
git archive --format=tar.gz HEAD \
    --prefix=apache-airflow-test/ \
    -o dist/apache-airflow-test-source.tar.gz
```